### ISSUE TITLE
Vocab.from_disk doesn't restore Token.pos

### ISSUE THREAD

**Issue by Jean-Zombie** [explosion/spaCy#2512]
## How to reproduce the behaviour
I instanciate spacy with the german model:
```python
nlp = spacy.load("de_core_news_sm")
>>> len(nlp.vocab)
77948
```
I process my strings and get the desired and expected output. The *pos* and *tag* info is present:
```python
doc = nlp("Dies ist ein kleiner Satz.")
doc.to_disk("saved_doc")
nlp.vocab.to_disk("vocab")
>>> len(doc.vocab), doc[0].orth_ , doc[0].pos_, doc[0].tag_
(77948, 'Dies', 'PRON', 'PDS')
```
When I load the vocab and doc-file again – as described [here](https://spacy.io/api/doc) – the *pos* info is gone. The vocab-size also got reduced somehow. 
```python
loaded_doc = Doc(Vocab().from_disk(vocab_path)).from_disk(doc_path)
>>> len(loaded_doc.vocab), loaded_doc[0].orth_, loaded_doc[0].pos_, loaded_doc[0].tag_
(77561, 'Dies', '', 'PDS')
```
Now strangely, if I load the saved vocab into `nlp.vocab` instead of `Vocab()` the *pos* info is restored correctly. But the vocab-size is now the sum of the size of the orginal `nlp.vocab` and the smaller `doc.vocab` – which seems kinda weird to me.
```python 
loaded_doc = Doc(nlp.vocab.from_disk(vocab_path)).from_disk(doc_path) # change in code is here
>>> len(loaded_doc.vocab), loaded_doc[0].orth_, loaded_doc[0].pos_, loaded_doc[0].tag_
(155509, 'Dies', 'PRON', 'PDS')
```

Am I confusing the internals of spacys vocabulary here or are the export/restore features bugged? 

## Your Environment

* **spaCy version:** 2.0.11
* **Platform:** Darwin-17.6.0-x86_64-i386-64bit
* **Python version:** 3.6.5
* **Models:** de_core_news_sm

**Comment by Jean-Zombie** [comment]
So no bug? ;-)

Is my "workaround" adviced, i.e. loading the vocab into `nlp.vocab`
```
Doc(nlp.vocab.from_disk(vocab_path)).from_disk(doc_path)
```
What about the increase in the vocabulary size?

**Comment by honnibal** [comment]
Sorry for not getting back to you about this.

The `pos` thing is expected (albeit confusing) behaviour. The tag-to-POS mapping is stored in the `vocab.morphology` class, so when you create a blank `Vocab()` to deserialize the doc, you don't have the tag mapping.

The vocab size thing was indeed a bug, now fixed in #2153. Thanks for reporting, and sorry again for not replying.
